storage/buckets/checkRequesterPays.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package buckets
+
+// [START get_requester_pays_status]
+import (
+	"context"
+	"log"
+
+	"cloud.google.com/go/storage"
+)
+
+// checkRequesterPays gets requester pays status.
+func checkRequesterPays(bucketName string) error {
+	// bucketName := "bucket-name"
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		return err
+	}
+	log.Printf("Is requester pays enabled? %v\n", attrs.RequesterPays)
```

Comment:
I'm not very sure, as in here we're printing value which we got, but this should probably be done through writer

![image](https://user-images.githubusercontent.com/26030071/72892957-e51b9e00-3d28-11ea-802e-807126f596a7.png)
![image](https://user-images.githubusercontent.com/26030071/72892987-f19ff680-3d28-11ea-8ef4-19a99c9e9432.png)